src/inject.js

```diff
@@ -99,6 +107,7 @@ async function handleRequest (req) {
 
 window[injectionName] = {
   isLiquality: true,
+  isMetaMask: true, // OpenSea wants this
```

Comment:
If we add this, we are going to be treated as metamask in all applications, and the liquality logo will no longer display. I think we should enquire with @alokm on this as i'm not sure this will be acceptable

Reply:
Without it whenever the "Metamask" button in Opensea is clicked, the user is redirected to https://metamask.io/download/